server.go

```diff
@@ -120,10 +124,13 @@ func main() {
 			if err != nil {
 				return err
 			}
+			feishuWebhook := feishuWebhook
```

Comment:
This line seems useless?
```suggestion
```

Reply:
This is a copy of the variable in the local scope, so that its original value from environment is not overwritten. 

There is a narrow use case where people may mix the two ways to specifying botUUID. The user may specify `FEISHU_WEBHOOK` env as the default bot, while also add uuid in the path for some of the grafana contacts.